src/electron/device-connect-view/Styles/deviceconnectview.scss

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+@import '../../../common/styles/common.scss';
+
+body {
+    -webkit-app-region: drag;
```

Comment:
Is there a technical limitation that requires we put this on `body` and then disable it per-child like the `button` CSS does? I'd have expected to want to put this on `.window-title` instead...

Reply:
Not a technical limitation, just my assumption about what the behavior should be. I will follow up with design about the intended behavior.

Reply:
Word back is, "When there's a lack of an app header bar, the common pattern is to have the whole screen area dragable." So I will plan to leave this as-is, unless you still have a concern.